lld/ELF/Arch/RISCV.cpp

```diff
@@ -1234,3 +1291,215 @@ TargetInfo *elf::getRISCVTargetInfo() {
   static RISCV target;
   return &target;
 }
+
+TableJumpSection::TableJumpSection()
+    : SyntheticSection(SHF_ALLOC | SHF_EXECINSTR, SHT_PROGBITS,
+                       config->wordsize, ".riscv.jvt") {}
```

Comment:
According to the RISC-V unprivileged specification (27.14.3. jvt CSR) the jvt section should be aligned to 64 bytes.
"If jvt is writable, the set of values the register may hold can vary by implementation. The value in the
BASE field must always be aligned on a 64-byte boundary."